README.md

```diff
@@ -19,10 +19,10 @@ For integration instructions, see our online docs: [docs.bugsnag.com/performance
 We are currently publishing pre-1.0 releases. These releases may contain breaking changes, which are documented in the [CHANGELOG.md](./CHANGELOG.md).
 ## Support
 
-* [Read the integration guide](https://docs.bugsnag.com/performance/cocoa/)
+* [Read the integration guide](https://docs.bugsnag.com/performance/ios/)
 * [Search open and closed issues](https://github.com/bugsnag/bugsnag-cocoa-performance/issues?utf8=✓&q=is%3Aissue) for similar problems
 * [Report a bug or request a feature](https://github.com/bugsnag/bugsnag-cocoa-performance/issues/new)
 
 ## License
 
-The BugSnag Cocoa Performance SDK is free software released under the MIT License. See the [LICENSE](https://github.com/bugsnag/bugsnag-cocoa-performance/blob/master/LICENSE) for details.
\ No newline at end of file
+The BugSnag Cocoa Performance SDK is free software released under the MIT License. See the [LICENSE](https://github.com/bugsnag/bugsnag-cocoa-performance/blob/master/LICENSE) for details.
```

Comment:
Git refuses to allow me to remove the newline on this file... not sure what's going on there.